lib/portage/data.py

```diff
@@ -178,6 +178,16 @@ def _get_global(k):
 				noiselevel=-1)
 			portage_group_warning()
 
+		portage_homedir_ready = \
+				os.access(os.path.expanduser("~portage"),
+						  portage.os.R_OK | portage.os.W_OK | portage.os.X_OK)
+		if not portage_homedir_ready:
+			writemsg(
+				colorize("BAD",
+				  _("portage: unable to utilise 'portage' user home directory."
```

Comment:
Please use the American English spelling of "utilize".

Reply:
No worries, I can certainly change it. I did not realise I'd used British English at the time but will keep an eye out for typos like this in the future.